test/model/GeneralizedHiddenMarkovModelTest.cpp

```diff
@@ -42,6 +42,9 @@
 // Tested header
 #include "model/GeneralizedHiddenMarkovModel.hpp"
 
+// Exception header
+#include "exception/NotYetImplemented.hpp"
```

Comment:
Same as in the other source files. Put this under `// ToPS headers`. As this file is a test for something within `namespace model`, then start with `model` headers, then `exception` headers and finally `helper` methods.